lib/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :title, :author, :publication_year
+
+  def initialize(data)
+    @title = data[:title]
+    @author = data[:author_first_name] + " " + data[:author_last_name]
+    @publication_year = data[:publication_date][-4..-1]
```

Comment:
I'd rather keep the whole date here, and then make a method that just pulls the year, that way I have access to more data if I need it.